cognite/client/_api/sequences.py

```diff
@@ -103,8 +103,8 @@ def __call__(
             external_id_prefix (str | None): Filter out sequences that do not have this string as the start of the externalId
             metadata (dict[str, str] | None): Filter out sequences that do not match these metadata fields and values (case-sensitive). Format is {"key1":"value1","key2":"value2"}.
             asset_ids (typing.Sequence[int] | None): Filter out sequences that are not linked to any of these assets.
-            asset_subtree_ids (int | typing.Sequence[int] | None): Asset subtree id or list of asset subtree ids to filter on.
-            asset_subtree_external_ids (str | SequenceNotStr[str] | None): Asset subtree external id or list of asset subtree external ids to filter on.
+            asset_subtree_ids (int | typing.Sequence[int] | None): return a list of assets that are in a subtree rooted at any of these assetIds.
+            asset_subtree_external_ids (str | SequenceNotStr[str] | None): Return a list of assets that are in a subtree rooted at any of these assetExternalIds.
```

Comment:
Similar comment for the other resource types:
```suggestion
            asset_subtree_ids (int | typing.Sequence[int] | None): return a list of sequences that are in a subtree rooted at any of these assetIds.
            asset_subtree_external_ids (str | SequenceNotStr[str] | None): Return a list of sequences that are in a subtree rooted at any of these assetExternalIds.
```